### DISCUSSION TITLE
How to add a new file to the current folder ?

### DISCUSSION THREAD

**Discussion by domellen** [chhoumann/quickadd#472] [Q&A]
Hello all,

I have a command to add a new file with a template to the inbox with QuickAdd (and Templater). It shows a form where I type the file name I want and it then adds this new file to the inbox using the template I chose. This works fine.

Is there a way to add this new file to the current folder and not to the inbox?  To be clear, let's say I'm looking at a file named "file 1" which is in the folder "folder 3". I would like to call QuickAdd from the Command palette and have this new file put into "folder 3", not into the inbox.

Thanks for any help.

**Comment by chhoumann** [comment]
Hi @domellen,

You can use the following settings in your Template Choice:
![image](https://github.com/chhoumann/quickadd/assets/29108628/b25622a3-92bd-46b7-8505-46c870000524)

That will add it to the same folder as the active file.

![image](https://github.com/chhoumann/quickadd/assets/29108628/2b2c9845-6c22-4955-bf7a-28150ff58331)
